addons/sourcemod/scripting/gokz-replays/recording.sp

```diff
@@ -493,120 +527,55 @@ static void WriteTickData(File file, int client, int replayType)
 {
     ReplayTickData tickData;
     ReplayTickData prevTickData;
-    int i;
     switch(replayType)
     {
         case ReplayType_Run:
         {
-            // 2 seconds pre timer starting
-            i = recordingIndexOnTimerStart[client] - preAndPostRunTickCount;
-            if (i < 0)
-            {
-                i = recordedTickData[client].Length - 1 + i;
-            }
-
-            int previousJ = i;
+            // Full run including pre and post
+            int previousJ = 0;
```

Comment:
Not really an issue with this PR as such, but this declaration and the isFirstTick can be taken outside the switch statement, they're common to all three cases

Reply:
Yep, good point!